composer.json

```diff
@@ -49,6 +50,10 @@
     "post-update-cmd": "Neos\\Flow\\Composer\\InstallerScripts::postUpdateAndInstall",
     "post-install-cmd": "Neos\\Flow\\Composer\\InstallerScripts::postUpdateAndInstall",
     "post-package-update": "Neos\\Flow\\Composer\\InstallerScripts::postPackageUpdateAndInstall",
-    "post-package-install": "Neos\\Flow\\Composer\\InstallerScripts::postPackageUpdateAndInstall"
+    "post-package-install": "Neos\\Flow\\Composer\\InstallerScripts::postPackageUpdateAndInstall",
+    "test-unit": "phpunit -c Build/BuildEssentials/PhpUnit/UnitTests.xml",
+    "test-functional": "phpunit -c Build/BuildEssentials/PhpUnit/FunctionalTests.xml",
+    "test-static": "psalm --config=Packages/Framework/psalm.xml --show-info=false",
+    "psalm-baseline-update": "psalm --config=Packages/Framework/psalm.xml --set-baseline=Packages/Framework/psalm-baseline.xml"
```

Comment:
This updates the baseline for the flow dev collection only. We don't have a baseline for the Neos dev collection yet?

Reply:
Nope, not yet. See https://github.com/neos/neos-development-collection/issues/2680